app-backend/datamodel/src/main/scala/AnnotationLabel.scala

```diff
@@ -137,33 +217,18 @@ final case class AnnotationLabelWithClassesProperties(
     annotationLabelClasses: List[UUID]
 )
 
-object AnnotationLabelWithClassesProperties {
-  implicit val annotationLabelWithClassesPropertiesEncoder
-    : Encoder[AnnotationLabelWithClassesProperties] =
-    new Encoder[AnnotationLabelWithClassesProperties] {
-      final def apply(
-          properties: AnnotationLabelWithClassesProperties
-      ): Json = {
-        (
-          Map(
-            "createdAt" -> properties.createdAt.asJson,
-            "createdBy" -> properties.createdBy.asJson,
-            "annotationProjectId" -> properties.annotationProjectId.asJson,
-            "annotationTaskId" -> properties.annotationTaskId.asJson,
-            "annotationLabelClasses" -> properties.annotationLabelClasses.asJson
-          )
-        ).asJson
-      }
-    }
-}
-
+@JsonCodec
 final case class AnnotationLabelWithClassesFeatureCollection(
```

Comment:
I _believe_ this can be delete too -- I tested it locally and everything compiles with it removed

Reply:
hmm, initially I was thinking that we might need it for one of the annotate frontend endpoints, but you're right - we're doing everything using List[AnnotationLabelWithClasses.GeoJSON] or List[AnnotationLabelWithClasses] directly.

Reply:
ooh, found a bunch of case classes in Annotation.scala that are no longer used either.
:scissors:

Reply:
Nice, I figured there were other ones too but didn't think to look in there as well